.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ orbs:
               name: Prepare build environment
               command: |
                 echo "export BUILD_NUMBER=`/home/circleci/repo/bin/generate-build-number`" >> $BASH_ENV
-                echo "export PATH=/home/circleci/.local/bin:$PATH" >> $BASH_ENV
+                echo 'export PATH=/home/circleci/.local/bin:$PATH' >> $BASH_ENV
```

Comment:
Did you intend to to send the variable to $BASH_ENV as opposed to expanding the existing path?